doc/ko/README-ko.md

```diff
@@ -0,0 +1,147 @@
+# Quantum Information Software Kit (QISKit)
+
+[![PyPI](https://img.shields.io/pypi/v/qiskit.svg)](https://pypi.python.org/pypi/qiskit)
+[![Build Status](https://travis-ci.org/QISKit/qiskit-sdk-py.svg?branch=master)](https://travis-ci.org/QISKit/qiskit-sdk-py)
+
+**QISKit**은 [OpenQASM](https://github.com/QISKit/qiskit-openqasm)과 [IBM Q experience (QX)](https://quantumexperience.ng.bluemix.net/)에서 사용할 수 있는 SDK(software development kit)입니다. 
+
+양자컴퓨팅 프로그램을 만들고, 컴파일하고, 실행하기 위해 **QISKit**을 사용하세요. 실행에 있어 QISKit은 온라인상 접근 가능한 실제 양자 프로세서, 온라인 시뮬레이터, 로컬 시뮬레이터의 백엔드 환경을 지원합니다. 온라인 백엔드 환경에서, QIAKit은 IBM Q experience와 연결하기 위해 [python API client](https://github.com/QISKit/qiskit-api-py)를 사용합니다.
```

Comment:
QIAKit --> QISKit?

Reply:
Oh.. It's my mistake. I changed it from recent commit